packages/cli/test/integration/mfa/mfa.api.test.ts

```diff
@@ -232,6 +260,28 @@ describe('Change password with MFA enabled', () => {
 	});
 });
 
+describe('MFA before enable checks', () => {
+	test('POST /can-enable should throw error if mfa.beforeSetup returns error', async () => {
+		externalHooks.run.mockRejectedValue(new BadRequestError('Error message'));
+
+		await testServer.authAgentFor(owner).post('/mfa/can-enable').expect(400);
+
+		expect(externalHooks.run).toHaveBeenCalledWith('mfa.beforeSetup', [
+			expect.objectContaining(owner),
+		]);
+	});
+
+	test('POST /can-enable should not throw error if mfa.beforeSetup does not', async () => {
```

Comment:
does not what? exists probably? :)
```suggestion
	test('POST /can-enable should not throw error if mfa.beforeSetup does not exist', async () => {
```